frontend/src/components/MyCredentialList.vue

```diff
@@ -57,6 +57,15 @@
         {{ item.issuedAt | moment("YYYY-MM-DD HH:mm") }}
       </template>
 
+      <template v-slot:[`item.revoked`]="{ item }">
```

Comment:
If I read this correctly you don't distinguish between a non-revoked (but revocable) credential and a credential that is in general non-revocable. I'd say we should let the user know if a credential is not revocable or non-revoked but revocable.

Reply:
I added a third state same as within the CredEx.vue (see screenshot above). By doing this I noticed that MyCredentialList has no concept of the credential exchange states. If the exchange gets stuck for whatever reason we would display it as valid here. I could change this by adding a filter to the backend that only shows acked credentials. Or totally change the behavior that we only create a my credential representation when we hit the acked state.

Reply:
I think right now we only need the acked state (all auto flags are on), so I will remove one of the handlers in the backend.

The other question is what will happen if the auto flags 

--auto-respond-credential-proposal
--auto-respond-credential-offer

are disabled, then one part of the flow would be in the wallet (upgrade document to credential) and the other in the partner details (accept credential).